app/server-utils/stampy.ts

```diff
@@ -263,6 +288,10 @@ export const loadQuestionDetail = withCache('questionDetail', async (question: s
     const {data} = await loadTags('NEVER_RELOAD')
     allTags = Object.fromEntries(data.map((r) => [r.name, r])) as Record<string, Tag>
   }
+  if (Object.keys(allBanners).length === 0) {
```

Comment:
isn't this fun...

Reply:
feel free to create an issue where you collect all notes about what needs to be double checked and loaded in correct order currently and deserves simplification, I might get to it during aisafety.com hackathon 🤞